src/core.js

```diff
@@ -1,7 +1,6 @@
-import get from 'lodash/fp/get';
+import * as R from 'ramda';
```

Comment:
Please use the named imports, instead of `* as R`.

Reply:
Fixed.